server/index.js

```diff
@@ -1,5 +1,6 @@
 const express = require("express");
 const compression = require("compression");
+const requestIp = require("request-ip");
```

Comment:
I was going through the readme of this package, and it seems like they don't look at `req.ip`, which seems to be recommended by express docs (https://expressjs.com/en/api.html#req.ip). any thoughts on whether we need to fallback to `req.ip`?

Reply:
Yes, makes sense!

Reply:
Actually i just realized, we don't need this fallback. Because `req.ip` itself comes from `X-Forwarded-For` header. Which is [appropriately considered by the package](https://www.npmjs.com/package/request-ip#how-it-works). Should i revert my change ?